lib/wibox/widget/slider.lua

```diff
@@ -318,6 +320,11 @@ function slider:set_value(value)
     if changed then
         self:emit_signal( "property::value", value)
         self:emit_signal( "widget::redraw_needed" )
+        if self._private.handle_widget then
+            self:emit_signal("widget::layout_changed")
+        elseif self._private.bar_widget then
+            self:emit_signal("widget::layout_changed")
+        end
```

Comment:
Both branches have the same body. Please make them be part of the same if statement.

Reply:
That was a desperate attempt at testing, Just fixed that.